test/integration/targets/pip/vars/main.yml

```diff
@@ -2,6 +2,12 @@ pip_test_package: sampleproject
 pip_test_packages:
   - sampleproject
   - decorator
+pip_test_pkg_ver:
+  - sampleproject<=100,!=9.0.0,>=0.0.1
```

Comment:
You might want to test with spaces in different places.

Reply:
I will fix these with some new test cases added together.

Reply:
so?